src/inference/emcee.jl

```diff
@@ -63,6 +63,15 @@ function AbstractMCMC.sample_end!(
     invlink!(spl.state.vi, spl)
 end
 
+function _typed_draw(model, vi::VarInfo)
+    spl = DynamicPPL.SampleFromPrior()
+    empty!(vi)
+    model(vi, spl)
+    DynamicPPL.link!(vi, spl)
```

Comment:
I'm wondering why `link!` is not imported (I think?) even though `invlink!` is. Or maybe we should always use qualified names for internal stuff from DynamicPPL?

Reply:
I'm not sure. Part of the reason all the DynamicPPL stuff is unqualified is because it was in Turing before they were separated, so I'm not sure what our go-forward style should be for this.